### ISSUE TITLE
Can't install 1.0.0

### ISSUE THREAD

**Issue by jethrogb** [rust-lang-nursery/rustup.rs#566]
```
$rustup toolchain add 1.0.0
info: syncing channel updates for '1.0.0-x86_64-unknown-linux-gnu'
error: could not download file from 'https://static.rust-lang.org/dist/channel-rust-1.0.0.toml.sha256'
info: caused by: http request returned an unsuccessful status code: 403
```
This file seems to only exist for 1.8.0 and up.

```
$rustup toolchain add stable-2015-05-15
info: syncing channel updates for 'stable-2015-05-15-x86_64-unknown-linux-gnu'
error: could not download file from 'https://static.rust-lang.org/dist/2015-05-15/channel-rust-stable.toml.sha256'
info: caused by: http request returned an unsuccessful status code: 403
```
There seems to only be a text version of this file.

**Comment by est31** [comment]
I've hit the same bug, when wanting to install rust 1.7.0. Its important to have access to historic versions. I for example want to test whether my project runs on rust 1.7.0 so that I can suggest my users to install the debian packages, which I would prefer over asking them to download multirust and stuff.

**Comment by brson** [comment]
rustup is supposed to fall back to other installation methods when it can't find the modern installation manifests, but the feature is broken. I *thought* a PR went through to fix it, but it definitely doesn't work on 0.2.0.

Duplicate of https://github.com/rust-lang-nursery/rustup.rs/issues/452
